### PR TITLE
Add shipping delivery days

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it adds shipping delivery days fields to shipping methods.

**PR intended to be tested with API branch:**  master

### Screenshots

Zrzut ekranu 2020-12-18 o 11 41 24

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Translated strings are extracted.
1. [x] Number of API calls is optimized.
1. [x] The changes are tested.
1. [x] Data-test are added for new elements.
1. [x] Type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1643 Add input fields to allow editing delivery days for shipping methods](https://app.clickup.com/t/2549495/SALEOR-1643)

**Review by krzysztofwolski — APPROVED** [review]

File: src/shipping/components/ShippingZoneInfo/ShippingZoneInfo.tsx [hunk 1]
```diff
@@ -1,30 +1,71 @@
...
 }
 
-const ShippingZoneInfo: React.FC<ShippingZoneInfoProps> = ({
-  data,
-  disabled,
-  errors,
-  onChange
-}) => {
+const ShippingZoneInfo: React.FC<ShippingZoneInfoProps> = props => {
+  const { data, hasDaysRange = false, disabled, errors, onChange } = props;
```
Comment (jwm0) [inline src/shipping/components/ShippingZoneInfo/ShippingZoneInfo.tsx:63]: Why not leave the destructuring inside the fn argument instead?

**Review by jwm0 — CHANGES_REQUESTED** [review]

File: src/shipping/components/ShippingZoneInfo/ShippingZoneInfo.tsx (hunk 1 above)
Comment (orzechdev) [reply src/shipping/components/ShippingZoneInfo/ShippingZoneInfo.tsx:63]: Because `props` might be passed in `const classes = useStyles(props);` further.

**Review by jwm0 — APPROVED** [review]

**Review by jwm0 — COMMENTED** [review]
Why is there an asterix `*` on the Max Delivery Time input? I checked and the field itself is not required. @pwgryglak you might have more insight on this

**Pull request merged**